src/clients/kafkaRest/models/ProduceRequestHeader.ts

```diff
@@ -24,7 +24,7 @@ export interface ProduceRequestHeader {
    * @type {string}
    * @memberof ProduceRequestHeader
    */
-  name: string;
+  name?: string;
```

Comment:
Minor fix to reduce the number of file changes in this PR:
```suggestion
  name: string;
```

Reply:
done!